crates/core/src/error.rs

```diff
@@ -103,6 +103,9 @@ pub enum SubscriptionError {
     SideEffect(Crud),
     #[error("Unsupported query on subscription: {0:?}")]
     Unsupported(String),
+    // TODO: Is this correct? Check if you can have multiple queries for the same table in a single call.
```

Comment:
You can have multiple queries for the same table. They should be added in distinct `SubscribeSingle` calls, but I suppose the legacy `Subscribe` can still add several of them at once. Historically we have imposed that they must be disjoint, but the new design as proposed includes client-side handling to allow overlapping queries.